_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
```

Comment:
Shouldn't this be still seamless to citizens, where they can't differentiate between the old and new systems, but only feel a continuous improvement in the experience?

Reply:
TBS uses whole-of-government perspective for services -- could be relevant from citizen's point-of-view too.